src/xcode/ENA/ENA/Source/SceneDelegate.swift

```diff
@@ -50,9 +50,17 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate, RequiresAppDepend
 		self.window = window
 
 		#if UITESTING
-		if let isOnboarded = UserDefaults.standard.value(forKey: "isOnboarded") as? String {
+		if let isOnboarded = UserDefaults.standard.string(forKey: "isOnboarded") {
 			store.isOnboarded = (isOnboarded != "NO")
 		}
+
+		if let onboardingVersion = UserDefaults.standard.string(forKey: "onboardingVersion") {
```

Comment:
Is this ever true?

Reply:
Not yet. This was our first approach and we think it will be useful once we write a UI test for the delta onboarding screen, so we left it in 🤓